Sources/APIClient.swift

```diff
@@ -495,16 +495,20 @@ class DefaultAuthAPIClient: AuthAPIClient {
         let u = endpoint.appendingPathComponent("/sso/wechat/callback")
         let queryItems = [
             URLQueryItem(name: "code", value: code),
-            URLQueryItem(name: "state", value: state)
+            URLQueryItem(name: "state", value: state),
+            URLQueryItem(name: "x_sdk", value: "ios")
         ]
         var urlComponents = URLComponents(
             url: u,
             resolvingAgainstBaseURL: false
         )!
         urlComponents.queryItems = queryItems
         var urlRequest = URLRequest(url: urlComponents.url!)
-        urlRequest.httpMethod = "GET"
-        urlRequest.setValue("application/json", forHTTPHeaderField: "content-type")
+        urlRequest.httpMethod = "POST"
+        urlRequest.setValue(
+            "application/x-www-form-urlencoded",
+            forHTTPHeaderField: "content-type"
+        )
```

Comment:
Can we put the query in the request body? See https://medium.com/@serge.works.io/swift-how-to-create-a-http-post-request-with-application-x-www-form-urlencoded-body-bfd9cd26d6d5

Reply:
I just noticed if we do this, we have to adjust how we get `x_sdk` in the server side.